### PR TITLE
aaces-specific shell 

### PR DESCRIPTION
**Pull request by amyecampbell**
The following pull request adds, fixes, and/or modifies:

1. Insert text here referencing the exact scripts that update
2. If possible, I will make sure that the pull request only updates one feature at a time
3. Add to this list if necessary

I performed the following prior to filing this pull request:

- [ ] Tested that my change does not break the analysis pipeline
- [ ] Ran a linter through my code
- [ ] Update environment.yml if my code introduces new packages.

Lastly, I will assign individuals to review my code and be proud of making the pull request!

### CODE REVIEW

**Comment by gwaygenomics** [comment]
need to merge upstream branch. it looks like commits from the last PR you merged #19 needs to be incorporated first.

File: aacesanalysis.sh [hunk 1]
```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+#exec &>hgsc_analysis.out 
+
+############################################
```
Comment (gwaygenomics) [inline aacesanalysis.sh]: I think you can delete all of the comments in this heading from line 5 to 22

File: aacesanalysis.sh [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+# ~~~~~~~~~~~~~~~~~~~~~
+# This section will determine which samples meet a specific inclusion criteria 
+# for use in downstream analyses
+# ~~~~~~~~~~~~~~~~~~~~~
+
+# We are using data from curatedOvarianData version 1.8.0
+# NOTE: The Mayo Clinic Data is not currently in curatedOvarianData.
+
+# Output the samples for each dataset that pass the inclusion criteria
+#
```
Comment (gwaygenomics) [inline aacesanalysis.sh]: extra #

File: aacesanalysis.sh [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+R --no-save --args $AACES_PATH 1.DataInclusion/Scripts/A.getInclusion.R  # (Table 1)
+
+# Output the common genes and the MAD (Median Absolute Deviation) genes to be 
+# used in developing moderated t score vectors and in clustering, respectively. 
+# This script will also output Venn diagrams for visualizing overlapping genes 
+# (Sup. Fig. S1) #NOTE: Bonome (GSE12672) is removed following the across
```
Comment (gwaygenomics) [inline aacesanalysis.sh]: can remove references to "Bonome"

File: aacesanalysis.sh [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+# ~~~~~~~~~~~~~
+# Output across dataset correlations for MAD genes
+# NOTE: common genes used in downstream analyses
+R --no-save --args $KMIN $KMAX $NSTARTS $SEED FALSE $NO_SHUFFLE "madgenes" \
+$DATASETS "GSE26712_eset" < 2.Clustering_DiffExprs/Scripts/A.run_kmeans_SAM.R
+
+# ~~~~~~~~~~~~~
+# k means & SAM (with common genes)
+# ~~~~~~~~~~~~~
+# Perform k means and SAM (Figure 1)
```
Comment (gwaygenomics) [inline aacesanalysis.sh]: can remove Figure Instances

**Comment by amyecampbell** [comment]
@gwaygenomics is this ok? You approved it but also those getInclusion changes which were intentional, but not initially included in this PR are included. There's just a couple of changes.

**Comment by gwaygenomics** [comment]
yep, go ahead and merge

**Pull request merged**